examples/7nodes/init.sh

```diff
@@ -28,6 +43,16 @@ while (( "$#" )); do
             consensus=clique
             shift
             ;;
+        --numNodes)
+            re='^[0-9]+$'
+            if ! [[ $2 =~ $re ]] ; then
+                echo "ERROR: numberOfNodes value must be a number"
+                usage
+            fi
+            numNodes=$2
```

Comment:
Since numNodes > 7 will fail in createPermissionedNodesJson(), I'd suggest adding a check for >7 nodes and exiting with an 'unsupported' error message.

Reply:
resolved